.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -1,38 +1,25 @@
 name: "🐞 Bug Report"
 description: "Report a bug if something isn't working as expected in Elementor"
-title: "🐞 Bug report: "
 labels: ["bug"]
```

Comment:
We talked about removing the label at this step and only after the triage will we tag the label that will automatically open a Jira ticket

Reply:
@arielk As long the the "feature request" template exist, this label remains.

When we move the feature request, this label will be removed.

Reply:
I think the "Feature Request" template will remain as it is today, just open a discussion instead of an issue.